src/ui/pages/IdentifierDetails/IdentifierDetails.scss

```diff
@@ -3,16 +3,112 @@
     margin-bottom: 0;
   }
 
-  .multi-sig-contact-list {
-    ion-grid,
-    ion-col {
-      padding: 0;
+  .card-block {
+    margin: 0.75rem 0 0 !important;
+  }
+
+  .creation-timestamp  {
+    .card-details-info-block-data {
+      font-size: 0.875rem;
+      font-weight: 500;
+      color: var(--ion-color-secondary);
+
+      .card-details-info-block-key {
+        display: block;
+        font-size: 1rem;
+        color: var(--ion-color-primary);
+      }
+    }
+  }
+
+  .member:not(:last-child) {
+    margin-bottom: 0.75rem;
+  }
+
+  .view-more-members {
+    --background: rgba(187, 187, 187, 0.5);
+    --border-radius: 0.5rem;
+    color: var(--ion-color-primary);
+    width: 100%;
+    display: block;
+    margin-top: 1.25rem;
+    --box-shadow: none;
+    --padding-top: 0.625rem;
+    --padding-bottom: 0.625rem;
+    font-size: 1rem;
+    --background-activated: rgba(187, 187, 187, 0.5);
```

Comment:
Please use color names defined in `styles/colors.scss` instead of hardcoding values. 

I believe this one should be something like `rgba(var(--ion-color-dark-grey-rgb), 0.5)`.